src/Network.py

```diff
@@ -53,20 +55,19 @@ def __del__(self):
     def buildNetwork(self, hasTeacher):
         """ Build out the policy/evaluation combo network
         """
-        numFilters = self.parameters['filters']
         with tf.variable_scope('inputs', reuse=tf.AUTO_REUSE) as _:
             self.input = tf.placeholder(
-                shape=[None, self.dim, self.dim, 3],
+                shape=[None] + self.dims + [3],
```

Comment:
Do we really support 3+ dimensions on self.dims?

Reply:
No, this is a good point.  Technically, the way this line is written, `shape(self.input)` can be of any arbitrary dimension, so this is fine, but later on in the code, we are not employing n-dimensional convolution in the conv block of the residual tower.  I will revert this line change and we can come back to n-dimensional input in a future release.

Reply:
Actually, this doesn't warrant a line change now that I think about it.  It should be fine as-is, we just don't yet support 3+ dimensions in `self.dims`.